src/app/components/feed/SaveFeed.js

```diff
@@ -233,25 +226,16 @@ const SaveFeed = (props) => {
   };
 
   // Error states that cause the save/edit button to disable
-  const discoverableNoLicense = discoverable && (
-    !academicLicense &&
-    !commercialLicense &&
-    !openSourceLicense
-  );
   const noTitle = title.length === 0;
-  const disableSaveButton = saving || discoverableNoLicense || noTitle || dataPoints.length === 0;
+  const disableSaveButton = saving || noTitle || dataPoints.length === 0;
 
   const handleSave = () => {
     setSaving(true);
     const licenses = [];
-    if (academicLicense) licenses.push(1);
-    if (commercialLicense) licenses.push(2);
-    if (openSourceLicense) licenses.push(3);
     const input = {
       name: title,
       description,
       saved_search_id: selectedListId,
-      tags,
       licenses,
       discoverable,
```

Comment:
We can delete `discoverable` from the GraphQL query to.

Reply:
I guess I didn't need either, since discoverable I think is false by default. I removed both lines and the page I think works as expected